src/Utils/AttributesUtils.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Utils;
+
+/**
+ * AttributesUtils class used for getting class and style from attributes.
+ */
+class AttributesUtils {
```

Comment:
I'm not sure about this name. Maybe is it better `GlobalStyleUtils`?

Reply:
Agree, or maybe a mix between the two: `StyleAttributesUtil`? :thinking:

Reply:
`StyleAttributesUtils` makes more sense to me, because the global part is automatically generated.